libs/reports_tasks.py

```diff
@@ -37,60 +41,125 @@ def create_heatmap_report(config, yesterday_csv, heatmap_file, column):
         np.save(heatmap_file, heatmap_grid)
 
 
-def create_daily_report(config):
+def process_csv_raw_data(camera_id: str, source_file: str, time_from: datetime, time_until: datetime):
+    if not os.path.isfile(source_file):
+        logger.warn(f"No data for day! [Camera: {camera_id}]")
+        return
+    objects_logs = {}
+    for hour in range(time_from.hour, time_until.hour):
+        objects_logs[hour] = {}
+    with open(source_file, newline='') as csvfile:
+        reader = csv.DictReader(csvfile)
+        for row in reader:
+            row_time = datetime.strptime(row["Timestamp"], "%Y-%m-%d %H:%M:%S")
+            if time_from <= row_time < time_until:
```

Comment:
Does this work as an implicit and? :astonished:

Reply:
Yes, its documented in https://docs.python.org/3/reference/expressions.html#comparisons